coalib/results/TextPosition.py

```diff
@@ -7,20 +7,35 @@
 class TextPosition:
 
     @enforce_signature
-    def __init__(self, line: (int, None)=None, column: (int, None)=None):
+    def __init__(self,
+                 line: (int, None)=None,
+                 column: (int, None)=None,
+                 text: (tuple, list, None)=None,
```

Comment:
but so is the case with line and column right? if the user doesn't give a text, it has no meaning? only when the text is specified does this have meaning